Header header

# Position of the robot
float64 lat
float64 lon

# Identifier for the road to cross

# Heading of road to cross
float64 road_heading

# Car vectors

